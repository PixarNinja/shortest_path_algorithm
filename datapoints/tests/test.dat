0: (-2.000000, 0.000000)
1: (0.000000, -1.000000)
2: (0.000000, -6.000000)
3: (-2.000000, -1.000000)
4: (-3.000000, 3.000000)
5: (2.000000, -4.000000)
6: (-4.000000, 0.000000)
7: (2.000000, 0.000000)
8: (3.000000, 0.000000)
9: (-1.000000, 5.000000)
