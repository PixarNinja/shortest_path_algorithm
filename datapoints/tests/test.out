6 0
0 3
3 1
1 2
2 5
5 8
8 7
7 9
9 4
4 6
